src/components/ListOrgs.vue

```diff
@@ -337,10 +381,52 @@ const showCode = async (selectedOrg) => {
   }
 };
 
+const onEditButtonClick = (event) => {
+  isEditModalEnabled.value = true;
+  currentEditOrgId.value = _get(event, 'id', null);
+};
+
+const closeEditModal = () => {
+  isEditModalEnabled.value = false;
+  currentEditOrgId.value = null;
+};
+
 const closeDialog = () => {
   isDialogVisible.value = false;
 };
 
+const updateOrgData = async () => {
+  isSubmitting.value = true;
+  console.log('Updating org data', localOrgData.value);
+  await roarfirekit.value
+    .createOrg(
+      activeOrgType.value,
+      localOrgData.value,
+      _get(localOrgData.value, 'testData', false),
+      _get(localOrgData.value, 'demoData', false),
+      currentEditOrgId.value,
+    )
+    .then(() => {
+      isSubmitting.value = false;
+      closeEditModal();
+      toast.add({
+        severity: 'success',
```

Comment:
I know this wasn't introduced by your PR, but taking the opportunity here: ideally we would use constants for these. 

Something like: 

```js
// src/constants/toastConfig.js`

export const TOAST_SEVERITIES = Object.freeze({ 
  SUCCESS: 'success', 
  ERROR: 'error',
  // etc.
})

export const TOAST_DEFAULT_LIFE_DURATION = 3000

Reply:
👍🏻 to @maximilianoertel's comment here.